aiida/cmdline/commands/cmd_rabbitmq.py

```diff
@@ -0,0 +1,386 @@
+        click.secho(f'Create `{queue}`... ', nl=False)
+        echo_response(response)
+
+
+@cmd_queues.command('delete')
+@click.argument('queues', nargs=-1)
+@with_client
+def cmd_queues_delete(client, queues):
+    """Delete existing queues."""
+    for queue in queues:
+        params = {'if-empty': True, 'if-unused': True}
+        response = client.request('queues/{virtual_host}/{queue}', {'queue': queue}, method='DELETE', params=params)
+        click.secho(f'Delete `{queue}`... ', nl=False)
+        echo_response(response, exit_on_error=False)
+
+
+@cmd_tasks.command('list')
+@with_manager
+@decorators.only_if_daemon_not_running()
+@click.pass_context
+def cmd_tasks_list(ctx, manager):
+    """List all active process tasks.
+
+    This command prints a list of process pk's for which there is an active process task with RabbitMQ. Since tasks can
+    only be seen when they are not currently with a daemon worker, this command can only be run when the daemon is not
+    running.
+    """
+    for pk in get_process_tasks(ctx.obj.profile, manager.get_communicator()):
+        echo.echo(pk)
+
+
+def get_active_processes() -> list[int]:
+    """Return the list of pks of active processes.
+
+    An active process is defined as a process that has a node with its attribute ``process_state`` set to one of:
+
+        * ``created``
+        * ``waiting``
+        * ``running``
+
+    :returns: A list of process pks that are marked as active in the database.
+    """
+    from aiida.engine import ProcessState
+    from aiida.orm import ProcessNode, QueryBuilder
+
+    return QueryBuilder().append(
+        ProcessNode,
+        filters={
+            'attributes.process_state': {
+                'in': [ProcessState.CREATED.value, ProcessState.WAITING.value, ProcessState.RUNNING.value]
+            }
+        },
+        project='id'
+    ).all(flat=True)
+
+
+def iterate_process_tasks(
+    profile: Profile, communicator: kiwipy.rmq.RmqCommunicator
+) -> collections.abc.Iterator[kiwipy.rmq.RmqIncomingTask]:
+    """Return the list of process pks that have a process task in the RabbitMQ process queue.
+
+    :returns: A list of process pks that have a corresponding process task with RabbitMQ.
+    """
+    from aiida.manage.external.rmq import get_launch_queue_name
+
+    launch_queue = get_launch_queue_name(profile.rmq_prefix)
+
+    for task in communicator.task_queue(launch_queue):
+        yield task
+
+
+def get_process_tasks(profile: Profile, communicator: kiwipy.rmq.RmqCommunicator) -> list[int]:
+    """Return the list of process pks that have a process task in the RabbitMQ process queue.
+
+    :returns: A list of process pks that have a corresponding process task with RabbitMQ.
+    """
+    pks = []
+
+    for task in iterate_process_tasks(profile, communicator):
+        try:
+            pks.append(task.body.get('args', {})['pid'])
+        except KeyError:
+            pass
+
+    return pks
+
+
+@cmd_tasks.command('analysis')
```

Comment:
to stay in line with the names of the other commands, perhaps this should be called `analyze` instead?

Reply:
Yeah, makes sense. Will change it

Reply:
done